runtime/drivers/sqlite/registry.go

```diff
@@ -109,6 +109,40 @@ func (c *connection) CreateInstance(_ context.Context, inst *drivers.Instance) e
 	return nil
 }
 
+// CreateInstance implements drivers.RegistryStore.
+func (c *connection) EditInstance(_ context.Context, inst *drivers.Instance) error {
+	// Override ctx because sqlite sometimes segfaults on context cancellation
+	ctx := context.Background()
+
+	// sqlite doesn't support maps need to convert to json and write as bytes array
+	env, err := mapToJSON(inst.Env)
+	if err != nil {
+		return err
+	}
+
+	now := time.Now()
+	_, err = c.db.ExecContext(
+		ctx,
+		"UPDATE instances set olap_driver = $1, olap_dsn = $2, repo_driver = $3, repo_dsn = $4, embed_catalog = $5, updated_on = $6, env = $7 "+
+			"where id = $8",
```

Comment:
nit: uppercase `SET` and `WHERE`, and move `updated_on` as the last item in list